webtau-http/src/main/java/com/twosigma/webtau/http/HttpHeader.java

```diff
@@ -24,6 +24,9 @@
 
 import static java.util.stream.Collectors.joining;
 
+/**
+ * Represents request and response <code>HTTP</code> header
```

Comment:
While this comment makes sense here, it does produce some confusing documentation in your screenshot because the header param to, for example, `http.get` is annotated as `Represents request and response HTTP header` which for a param of that method is not correct and confusing.

Reply:
Idea was to describe type and not the parameter. But I see how it can be confused in the context of it usage. 
So I wonder if I should implement znai code reference feature first. I.e. if you hover over `HttpHeader` within the code it will popup its java doc, and if you click then it will navigate you to the reference page.

Reply:
I get the comment on the type but for someone who isn't a webtau developer and is just looking at the docs, it'll be super confusing.